nodebuilder/tests/sync_test.go

```diff
@@ -265,54 +256,52 @@ Steps:
 3. Check BN is synced to height 20
 4. Create a Full Node(FN) with a connection to BN as a trusted peer
 5. Start a FN
-6. Check FN is synced to height 30
-7. Create a Light Node(LN) with a connection to BN, FN as trusted peers
+6. Check FN is synced to network head
+7. Create a Light Node(LN) with a connection to BN and FN as trusted peers
 8. Start LN
-9. Check LN is synced to height 50
+9. Check LN is synced to network head.
 */
 func TestSyncLightWithTrustedPeers(t *testing.T) {
```

Comment:
It's now the same as TestSyncLightWithTrustedPeers from what it tests perspective or do I miss something?